articles/WideStrings.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: default
+title: WideStrings
+abstract:
+  Some ROS2 users would like to use multi-byte characters for topic data, suitable for supporting multi-byte languages.
```

Comment:
The combination of title and abstract is rather confusing. The main point is that you don't need to use `wstring` to send unicode. So maybe changing the title to mention both `string` as well as `wstring` would make sense.

Reply:
I made the title and abstract about unicode support rather than just wide strings. 4102e8e9bc44f4b7c73de2ce89dfbdbcb21f358b